recipes/libcugraph/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "libcugraph" %}
+{% set version = "0.19.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/rapidsai/cugraph/archive/refs/tags/v{{ version }}a.tar.gz
+    sha256: eea8eb788ee2e9f369b6d4911c8cb7c4819453e17aa855eac14af86ee27f627e
+    folder: cugraph
+  # libcugraph depends on the RMM header-only library of the same version.
+  - url: https://github.com/rapidsai/rmm/archive/refs/tags/v{{ version }}a.tar.gz
+    sha256: aa2b6542e51e9d3bec1c4053ab2a06af8fb8e9a94be3832418109904ebebb276
+    folder: rmm
+
+build:
+  number: 0
+  skip: true  # [(not linux) or (cuda_compiler_version == "None")]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
+  missing_dso_whitelist:
+    - "*/libcuda.*"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake >=3.12.4
+    - make
+    - libgomp
+  host:
+    - boost-cpp >=1.66
+    - nccl >=2.8.4
+    - libfaiss =*=*_cuda
+    - spdlog =1.7.0
+    - libgomp
+  run:
+    - libgomp
+
+test:
+  commands:
+    # Only testing for select headers and the library
+    - test -f $PREFIX/include/cugraph/version_confg.hpp
```

Comment:
Looks like this isn't being found on CI. Not sure if it is a typo (`version_config.hpp`?) or if this file doesn't exist

```
+ test -f /home/conda/staged-recipes/build_artifacts/libcugraph_1618613730358/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/include/cugraph/version_confg.hpp
Tests failed for libcugraph-0.19.0-h1a5f58c_0.tar.bz2 - moving package to /home/conda/staged-recipes/build_artifacts/broken
```

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=305041&view=logs&j=232b7f5e-8d64-5438-fada-8478f379d226

Reply:
Both :)  It looks like the v0.19.0a tarfile the recipe is pulling down is quite old (Jan 27). I guess when 0.19 is released, RAPIDS will build newer ones, and I'll have to update the hash?

Reply:
I added a temporary workaround and a `FIXME` describing that the workaround should be removed when the cugraph sources being downloaded are more current.